lib/common-types.ts

```diff
@@ -909,6 +909,34 @@ export interface Adapter {
   makeRequest: MakeRequestWithUserAgent
 }
 
+export type OpPatch = AddPatch | RemovePatch | ReplacePatch | MovePatch | CopyPatch | TestPatch
```

Comment:
Just a nit: 
By [definition](https://datatracker.ietf.org/doc/html/rfc6902/), a **Patch Document** is a list of **Operations**. Operations can have different Operation types. Operation can be mixed within a Patch.